Lib/test/test_winapi.py

```diff
@@ -161,13 +161,16 @@ def test_event_source_registration(self):
         source_name = "PythonTestEventSource"
 
         handle = _winapi.RegisterEventSource(None, source_name)
+        self.addCleanup(_winapi.DeregisterEventSource, handle)
         self.assertNotEqual(handle, _winapi.INVALID_HANDLE_VALUE)
 
-        with self.assertRaisesRegex(OSError, '[WinError 87]'):
+        with self.assertRaises(OSError) as cm:
             _winapi.RegisterEventSource(None, "")
+        self.assertEqual(cm.exception.winerror, 87)
```

Comment:
This is ERROR_INVALID_PARAMETER. You can define `ERROR_INVALID_PARAMETER = 87` or just add a comment:

```suggestion
        self.assertEqual(cm.exception.winerror, 87)  # ERROR_INVALID_PARAMETER
```
Alternatively, you can test `errno`:
```suggestion
        self.assertEqual(cm.exception.errno, errno.EINVAL)
```